src/status_im/ui/screens/wallet/settings/views.cljs

```diff
@@ -58,7 +58,7 @@
 (defview settings-hook []
   (letsubs [{:keys [label view]} [:get-screen-params :wallet-settings-hook]
             {address :address} [:account/account]]
-    [react/view {:style {:flex 1 :background-color colors/white}}
+    [react/keyboard-avoiding-view {:style {:flex 1 :background-color colors/white}}
```

Comment:
This one can't be purely done in the extension, right?

Reply:
right, unfortunately

Reply:
we can think later, how we can improve this